TM-3828: CM API documentation has wrong info and is insufficient (#190)
PiwikPRO/PPMS-PublicDocs: consent_manager/js_api/installation.rst

Installation
------------

> Consent Manager is fully integrated with Tag Manager. If you have already installed asynchronous snippet and you are using API only from Tag Manager tags, you are able use JavaScript API without aby pitfalls.

Author: fixed

> Only one thing should be considered before using API is where you call commands. If your goal is to perform API method outside Tag Manager tags like in below example:

Reviewer: ```suggestion
The one thing that should be considered before using API is where you call commands. Lets assume that your goal is to perform API method outside Tag Manager tags like in below example:
```

Author: what do you think about changing order of `below example` to `example below` ?
```suggestion
The one thing that should be considered before using API is where you call commands. Lets assume that your goal is to perform API method outside Tag Manager tags like in example below:
```

Author: fixed

.. code-block:: html

    <!DOCTYPE html>
    <html>
        <head>
        <meta charset="UTF-8">
        <title></title>
    </head>

    <!-- Start Piwik PRO Tag Manager code -->
    <script>
        // Tag Manager async code snippet
    </script>
    <!-- End Piwik PRO Tag Manager code -->

    <script>
        // API call outside Tag Manager injected manually
        ppms.cm.api(command, ...args);
    </script>

    <body>
        Rest content of document
    </body>

    </html>

> When you need execute API in such manner, you should take care about Tag Manager snippet version.

Author: fixed

> Because `ppms.cm.api` global object is initialized in snippet and/or in Tag Manager container, if you are using old version of Tag Manager snippet (PPMS version > 6.2), `ppms.cm.api` might be undefined until container will be loaded.

Reviewer 2: Shouldn't it be `PPMS version < 6.2`? I'm guessing based on context.

Author: correct, fixed

> So if you want use own scripts outside Tag Manager, you need update snippet to use `ppms.cm.api`, or use `dataLayer` interface if replacing snippet is not possible:

Reviewer: ```suggestion
So if you want to use your own scripts outside Tag Manager, you need to update the snippet to use `ppms.cm.api`, or use `dataLayer` interface if replacing snippet is not possible:
```

Reviewer 2: I would drop "So" from the beginning of sentence.

Author: fixed
